deploy/kubernetes/setup.sh

```diff
@@ -4,7 +4,7 @@ set -e
 usage() {
   echo
   echo 'Usage:'
-  echo '  setup.sh [-c collector-name] [-k cluster-name] [-n namespace] <endpoint> <access-id> <access-key>'
+  echo '  setup.sh [-c collector-name] [-k cluster-name] [-n namespace] [-a isAlpha] <endpoint> <access-id> <access-key>'
```

Comment:
nit: rather than `isAlpha` can we call this `useAlpha`?

Also:  In this same PR can we update the deployment README with the new parameter: https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/master/deploy#automatic-source-creation-and-setup-script